tools_docker/Debian_Uwsgi/Dockerfile

```diff
@@ -86,18 +86,15 @@ ENV LANGUAGE C.UTF-8
 ENV LC_ALL C.UTF-8
 
 RUN \
-    printf "Package: pushpin\nPin: version 1.30.*\nPin-Priority: 999" > /etc/apt/preferences.d/pushpin \
+    printf "Package: pushpin\nPin: version 1.31.*\nPin-Priority: 999" > /etc/apt/preferences.d/pushpin \
     && apt install -qy \
         apt-transport-https \
         software-properties-common \
         gnupg2 \
-        # NOTE - S.G. - 2020-08-10 - Installing zurl before adding pushpin repository
-        # as the zurl package in it cannot be installed on stretch.
-        zurl \
```

Comment:
Zurl no more used ?
If yes, we need to modify common.sh also because sed command exist for zurl and we create specific log file.

Reply:
`zurl` is still used, it's just that the version fanout provides in their repository is now installable on stretch. I'll have a look whether this changes the log file name.

Reply:
great !

Reply:
See below, the log file is unchanged so no change is needed in `common.sh`.